src/main/java/calculator/NumberCalculator.java

```diff
@@ -0,0 +1,12 @@
+package calculator;
+
+import java.util.List;
+
+public class NumberCalculator {
+
+	public static int sum(List<Integer> numbers) {
+		return numbers.stream()
+			.mapToInt(number -> number)
```

Comment:
```suggestion
			.mapToInt(Integer::intValue)
```
이렇게 사용 할 수 도 있으니 참고만 해주세요. :)